### ISSUE TITLE
downgrading

### ISSUE THREAD

**Issue by mspykerez** [jliddev/WowUp#16]
Allow us to manually downgrade an addon to an older version of choice. Some problems can occur on newer versions of an addon whereas with an older version it works just fine.

**Comment by Evolutious** [comment]
Agreed. Having a 'Version" context item in the right click menu with a list of released files would be awesome so that we could manually install a certain (previous version).
